ccan/daemonize/daemonize.c

```diff
@@ -6,7 +6,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 
-/* This code is based on Stevens Advanced Programming in the UNIX
+/* This code is based on Steven's Advanced Programming in the UNIX
  * Environment. */
```

Comment:
The author's name is "Richard Stevens", not "Richard Steven", so it should be "Stevens'", not "Steven's".

Reply:
Thanks. Fixed.

Reply:
Fixed in 53688a0b